packages/components/src/border-box-control/stories/index.story.tsx

```diff
@@ -16,7 +16,7 @@ import Button from '../../button';
 import { BorderBoxControl } from '../';
 
 const meta: Meta< typeof BorderBoxControl > = {
-	title: 'Components (Experimental)/BorderBoxControl',
+	title: 'Components/BorderBoxControl',
```

Comment:
I think we need to add this:

```diff
diff --git a/packages/components/src/border-box-control/stories/index.story.tsx b/packages/components/src/border-box-control/stories/index.story.tsx
index 7ac8cbbea8c..5341dacab64 100644
--- a/packages/components/src/border-box-control/stories/index.story.tsx
+++ b/packages/components/src/border-box-control/stories/index.story.tsx
@@ -83,4 +83,5 @@ export const Default = Template.bind( {} );
 Default.args = {
        colors,
        label: 'Borders',
+       enableStyle: true,
 };
```

without it, Storybook will display the control as `false` by default:


https://github.com/user-attachments/assets/d956b80b-0423-445c-879f-3373846a19f3

Reply:
Yeah, that's a good idea. Better default 👍